### DISCUSSION TITLE
Installing lunarvim fails today; same command worked recently

### DISCUSSION THREAD

**Discussion by dstromberg** [LunarVim/LunarVim#3707] [Q&A] [answered]
When I run:
`LV_BRANCH='release-1.2/neovim-0.8' bash <(curl -s https://raw.githubusercontent.com/lunarvim/lunarvim/master/utils/installer/install.sh)
`
I get:
```
Installing packer
Cloning into '/home/dstromberg/.local/share/lunarvim/site/pack/packer/start/packer.nvim'...
Lazy setup complete
--------------------------------------------------------------------------------
Verifying core plugins
mismatch at [packer.nvim]: expected [6afb674], got [b134cba]
[ERROR]: Unable to verify plugins, make sure to manually run ':Lazy sync' when starting lvim for the first time.
```.
..and I don't know why.

Does Lunarvim suddenly add its own version of packer or something?

I'm installing packer myself using:
```
  -- Set up packer. Use a protected call so we don't error out on first use
  local status_ok, packer = pcall(require, "packer")      
  if not status_ok then                                                              
      return                                                                         
  end                                                                                
                                                                                     
  -- Install plugins here                                                            
  return packer.startup(function(use)                                                
      -- My plugins here                                                             
      use {                                                                          
          'wbthomason/packer.nvim' -- Have packer manage itself                      
      }
```

Any suggestions folks?

Thanks!

**Comment by brneor** [comment, marked as answer]
Looks like there's some mixing in your install command: you're setting `LV_BRANCH` to a release branch and then using the script from master (which does not use `packer` anymore since yesterday). The correct install command should be 

```
LV_BRANCH='release-1.2/neovim-0.8' bash <(curl -s https://raw.githubusercontent.com/lunarvim/lunarvim/fc6873809934917b470bff1b072171879899a36b/utils/installer/install.sh)
```

according to [the docs](https://www.lunarvim.org/docs/installation).

**Reply by dstromberg** [reply]
That fixed it. Thanks!

**Reply by brneor** [reply]
Nice! Can you mark the question as answered?

**Reply by alfaruqii** [reply]
thankyou so much man! you make my day
